lib/parallel_tests/cli.rb

```diff
@@ -274,7 +274,11 @@ def parse_options!(argv)
 
       files, remaining = extract_file_paths(argv)
       unless options[:execute]
-        abort "Pass files or folders to run" unless files.any?
+        unless files.any?
```

Comment:
should also be `if files.empty?` ?

Reply:
I kept `unless files.any?` because that's the style that was used already :man_shrugging:..

Reply:
yeah, not your fault, but might as well change it ... positive logic is simpler :)

Reply:
Sure, let me update it!